addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.js

```diff
@@ -24,6 +24,24 @@ export class ThreadViewTopbar extends Component {
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+     * Determine whether description should display on top bar.
+     *
+     * @returns {boolean}
+     */
+    get hasThreadViewTopbarDescription() {
+        if (!this.threadViewTopbar.thread) {
+            return false;
+        }
+        return (
+            this.threadViewTopbar.thread.description ||
+            this.threadViewTopbar.thread.isChannelDescriptionChangeable &&
+            this.messaging.currentUser &&
+            !this.messaging.currentUser.isPortalUser &&
+            !this.messaging.isCurrentUserGuest
```

Comment:
Would include this:
```
this.messaging.currentUser &&
!this.messaging.currentUser.isPortalUser &&
!this.messaging.isCurrentUserGuest
```

In the definition of field `isChannelDescriptionChangeable`.
So that the condition is easier: it just shows description when there's actually a description or the description can be changed (by current user).